### ISSUE TITLE
[Q] Batch Normalization의 목적과 기능

### ISSUE THREAD

**Issue by skdy33** [YBIGTA/DeepNLP-Study#18]
BN은 internal covariate shift를 잡기 위하여.

근데 이 레이어가 성능을 개선시키기 위해서도 많이 사용되나?

**Comment by dreamgonfly** [comment]
Batch Norm 논문에서 가져왔습니다.

> When training with Batch Normalization, a training example is seen in conjunction with other examples in the mini-batch, and the training network no longer producing deterministic values for a given training example. In our experiments, we found this effect to be advantageous to the generalization of the network. Whereas Dropout is typically used to reduce over-fitting, in a batch-normalized network we found that i can be either removed or reduced in strength.

Dropout이 ensemble 역할을 해서 overfitting을 막고 성능을 개선하는 것처럼, Batch Norm 역시 한 training example을 볼 때 다른 examples를 참고하므로 overfitting을 막을 수 있는 것 같습니다.
